pages/static-security-analysis.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Static Security Analysis
+permalink: /static-security-analysis/
+---
+
+Static analysis is an important part of the development process, and is required for ATO. There are two main types of static security testing that needs to be done:
+
+1. Dependency analysis, where the ruby gems, python modules, and js packages your app uses are checked against a list of known vulnerabilities; and
+1. Code analysis in which your code is checked against a list of antipatterns.
+
+There are tools for JS, Ruby, and Python, and you are encouraged to set up this scanning early on in the development cycle to prevent unexpected delays when it's time to get your ATO.
+
+## Javascript
+
+### Dependency analysis
+
+We recommend using the Node Security Project's [NSP tool](https://nodesecurity.io/tools) to scan your package.json or npm-shrinkwrap.json file for known security issues.
+
+To install:
+
+    $ npm install nsp --global
+
+To run, from your project directory:
+
+    $ nsp check
+
+### Static code analysis
+
+There are several tools available for running analysis on JS projects, but the most full featured seems to be [ESLint](https://eslint.org). ESLint doesn't offer security scanning out of the box, but it is pluggable and Mozilla has provided a set of rules that mimic the behavior of their now deprecated ScanJS. These rules are available on [github](https://github.com/mozfreddyb/eslint-plugin-scanjs-rules).
+
+To install:
+
+    $ npm install eslint
+    $ npm install eslint-plugin-scanjs-rules
+
+To scan, from your project directory (for now, you can use the eslintrc file found [here](https://github.com/mozfreddyb/eslint-config-scanjs/blob/master/.eslintrc)):
+
+    $ eslint -c <PATH_TO_ESLINTRC> .
+
+## Ruby
+
+### Dependency analysis
+
+We recommend using [Bundler Audit](https://github/com/rubysec/bundler-audit) to scan your gems for known vulnerabilities. There are other tools available, most notably Gemnasium, but we are currently suggesting Bundler Audit until we can evaluate whether Gemnasium will work as a replacement for all of our languages.
```

Comment:
One more note about Gemnasium: if we wanted to get a snapshot of dependency health across all 18F repos, it would be as easy as visiting https://gemnasium.com/18f, or placing badges on a site like the Project Dashboard: https://project-dashboard.18f.gov/default